.gitignore

```diff
@@ -9,6 +9,7 @@ yarn.lock
 test/fixtures/ls-output/xxx.txt
 .nyc_output/
 .vscode/
+tsconfig.local.json
```

Comment:
There is a way to do it without having to include all those local `json` file in gitignore: 
In your project dir you'll have `.git/info/exclude` file and you can include it there like: 
```
# git ls-files --others --exclude-from=.git/info/exclude
# Lines that start with '#' are comments.
# For a project mostly in C, the following would be a good set of
# exclude patterns (uncomment them if you want to use them):
# *.[oa]
# *~

# vscode local settings directory
.vscode

#tests output
.nyc_output

tsconfig.local.json
```

Reply:
Just a suggestion though, if we see a benefit of including in gitignore, why not, this way we do it for the whole team as well

Reply:
Thanks for the new knowledge! Personally I prefer widely used files to be in gitignore, but will definitely remember in the future 💯